tmt-app/src/Styling/Home.css

```diff
@@ -12,8 +12,8 @@
     Home page Buttons Stlying
```

Comment:
Typo, but ideally would like to see us shift away from using .css files and classNames to define styles, and shift towards re-usable components or variants instead